docs/installation.rst

```diff
@@ -71,19 +71,21 @@ If you are on Fedora, install the build dependencies with this command:
 
 .. code-block:: bash
 
-   sudo dnf install -y gcc make cmake bzip2-devel expat-devel file-devel glib2-devel libcurl-devel libmodulemd-devel libxml2-devel python3-devel python3-gobject python3-libmodulemd rpm-devel openssl-devel sqlite-devel xz-devel zchunk-devel zlib-devel
+   sudo dnf install -y gcc make cairo-devel cairo-gobject-devel cmake bzip2-devel expat-devel file-devel glib2-devel gobject-introspection-devel libcurl-devel libmodulemd-devel libpq-devel libxml2-devel openssl-devel python3-devel rpm-devel sqlite-devel xz-devel zchunk-devel zlib-devel
```

Comment:
What is `libpg-devel` for, do we know?  That one is a little weird, it's postgresql related.

Reply:
yes it is posgres-devel package, for some reason pulpcore installation doesn't solve (for me) pip installation issue saying ``missing pg_config executable``

Reply:
So is it a pulpcore problem?  Or is this something the rpm plugin is somehow pulling in?

Reply:
It occurs already in pulpcore installation part so it is not an rpm issue. I removed it from our doc. It should be noted in core docs when pip error appears.